### PR TITLE
feat: implement the functionality of kubeadm reset using a script

### PR DESCRIPTION
**Pull request by OrangeBao**
#### What type of PR is this?

/kind feature

#### Which issue(s) does this PR fix?

Fixes #
#### Does this PR introduce a user-facing change?

```release-note

```

### CODE REVIEW

**Comment by OrangeBao** [comment]
When the node returns to the main cluster, the manifest directory will be restored.

File: hack/k8s-in-k8s/kubelet_node_helper.sh [hunk 1]
```diff
@@ -9,10 +9,115 @@ JOIN_HOST=$2
...
+    for file in "${files_to_delete[@]}"; do
+        echo "Deleting file: $file"
+        rm -rf $file
+    done
+}
+
+
+# similar to the reset function of kubeadm. kubernetes/cmd/kubeadm/app/cmd/phases/reset/cleanupnode.go
+function node_reset() {
+    echo "exec node_reset(1/5): stop kubelet...."
```
Comment (duanmengkk) [inline hack/k8s-in-k8s/kubelet_node_helper.sh]:
```suggestion
    echo "exec node_reset(1/4): stop kubelet...."
```

File: hack/k8s-in-k8s/kubelet_node_helper.sh [hunk 2]
```diff
@@ -9,10 +9,115 @@ JOIN_HOST=$2
...
+    systemctl stop kubelet
+
+    echo "exec node_reset(2/5): remove container of kubernetes...."
```
Comment (duanmengkk) [inline hack/k8s-in-k8s/kubelet_node_helper.sh]:
```suggestion
    echo "exec node_reset(2/4): remove container of kubernetes...."
```

File: hack/k8s-in-k8s/kubelet_node_helper.sh [hunk 3]
```diff
@@ -9,10 +9,115 @@ JOIN_HOST=$2
...
+    else 
+        echo "Unknown runtime: $CRI_SOCKET"
+        exit 1
+    fi
+    
+    echo "exec node_reset(3/5): unmount kubelet lib...."
+    # /kubernetes/cmd/kubeadm/app/cmd/phases/reset/cleanupnode.go:151 CleanDir
+    unmount_kubelet_directory "${PATH_KUBELET_LIB}"
+
+    echo "exec node_reset(3/5): clean file for kubernetes...."
```
Comment (duanmengkk) [inline hack/k8s-in-k8s/kubelet_node_helper.sh]:
```suggestion
    echo "exec node_reset(4/4): clean file for kubernetes...."
```

**Pull request merged**